registry-requirements.md

```diff
@@ -13,112 +12,131 @@ From the charter:
-
-Aspects to clarify based on the analysis of other registries above:
-
-- Versioning of registry entries (see https://github.com/w3c/wot/tree/main/registry-analysis#versioning) and versioning with respect to 
-- Deletion and deprecation (see https://github.com/w3c/wot/tree/main/registry-analysis#deletion-and-deprecation-of-registry-entries)
-- Differentiating entry into the registry and update
-- Technical submission mechanism. How does a binding get submitted? Is it a PR, an issue linking to an existing document, or an email? See the submission mechanism fields above.
-- Whether we should have one or multiple registries
-
-### Registry Template
-
-Submission template vs. what lives in the registry entry
-
-A set of files and requirements for each file
-
-At the top level, markdown file similar to the use case template that describes the binding, YAML template for git workflow 
-
-- name of binding
-- maintainers
-- protocols
-
-Same content as existing binding
-- HTML specification file using respec
-- JSON Schema
-- Ontology files including JSON-LD context for TDs
-
-(Think about how to manage the set of semantic artifacts/ontologies)
-
-Create a repository template to make it easy for people to create new bindings
+1. A binding that uses a protocol MUST map at least one WoT operation (`op` keyword value such as `readproperty`) to a protocol message and vice versa
+2. A binding that uses a serialization format via the `contentType` keyword MUST mention how the Data Schema terms should be used to describe the messages. This avoids submission of a binding like "XML Binding" that says "Use `contentType:application/xml` and nothing more. That alone would not be enough to serialize correct messages based on the data schema.
+  - TODO: We will need additional mechanisms (including vocabulary terms) to ensure that it is possible to use other media types.
+3. Initial entry MUST be a correct document which complies with point 10 (document content). The reviewer MUST NOT check whether the binding tries to map `readproperty` to a non-existent HTTP method. A successful initial document triggers a "Call for Implementation".
+4. The WoT binding MAY be just one section of the document. In that case, the "Link to the binding document" in the registry entry MUST point to the specific location. PDF or similar document types MAY be submitted if the "Link to the binding document" in the registry entry contains a text pointing to the section. However, HTML and Webpages SHOULD be favoured.
+5. The WoT binding document DOES NOT have to follow the W3C copyright. The submitter is free to choose based on the process they or their organization follows.
+6. The binding document linked in the registry entry SHOULD be open to read, use, and implement, but it is not required for the document to be added to the registry.
```

Comment:
```suggestion
6. The binding document linked in the registry entry SHOULD be open to read, use, and implement, but it is not required that the document be added to the registry.
```

Reply:
I think both proposals are changing the meaning. We want to say the openness is not a requirement but is only recommended

Reply:
Maybe this change could already clarify the bullet point?

```suggestion
6. The binding document linked in the registry entry SHOULD be open to read, use, and implement, but that is not required for the document to be added to the registry.
```

Reply:
I did not understand the intended meaning. Have revised my suggestion (to match that from @JKRhb), now that I *think* I understand the intent.

Reply:
I have used the suggestion from @TallTed with the revision in the end as that was the initial discussion start. Thanks for the suggestions both!